wiki/Tournaments/CIC/2012/en.md

```diff
@@ -0,0 +1,306 @@
+- Tiebreaker
+  - **[IOSYS - DUGEM (lufi10) \[Dugem Overdose!!\]](https://osu.ppy.sh/beatmapsets/51008#fruits/158770)**
+
+### Second Round
+- NoMod
+  - [dj TAKA - V -conclusion- (alvisto) \[Another\]](https://osu.ppy.sh/beatmapsets/11674#fruits/45360)
+  - [Sound Horizon - Raijin no Hidariude (AngelHoney) \[Insane\]](https://osu.ppy.sh/beatmapsets/16792#fruits/60089)
+  - [Silver Forest - Phantasm Brigade (S i R i R u) \[Lunatic\]](https://osu.ppy.sh/beatmapsets/11543#fruits/44031)
+  - [Shihoko Hirata - Signs of Love (Reincarnation ver.) (Louis Cyphre) \[Insane\]](https://osu.ppy.sh/beatmapsets/38700#fruits/123824)
+  - [Seiryu X Donald - Time to Donald (James) \[Another\]](https://osu.ppy.sh/beatmapsets/6950#fruits/30613)
+  - [Infinite Stratos Cast - SUPER STREAM \~INFINIT FUTURE Revised Mix\~ (JauiPlaY) \[DisPlaY\]](https://osu.ppy.sh/beatmapsets/29601#fruits/98163)
+  - [07th Expansion - Final Answer (gowww) \[Insane\]](https://osu.ppy.sh/beatmapsets/26226#fruits/88633)
+  - [xi - Halcyon (gowww) \[Another\]](https://osu.ppy.sh/beatmapsets/20871#fruits/73699)
+  - [Hatsune Miku - Hidari Migi -migihidari- (NatsumeRin) \[Rin\]](https://osu.ppy.sh/beatmapsets/24177#fruits/87223)
+  - [GIRL NEXT DOOR - Be your wings (blissfulyoshi) \[vs\]](https://osu.ppy.sh/beatmapsets/8518#fruits/35566)
+- Tiebreaker
+  - **[Kagamine Rin - Oni Kanojo (val0108) \[gow's Insane\]](https://osu.ppy.sh/beatmapsets/23616#fruits/86245)**
+
+### First Round
+- NoMod
+  - [SCANDAL - Shunkan Sentimental (TV Size) (saymun) \[Breeze's Difficulty\]](https://osu.ppy.sh/beatmapsets/12756#fruits/48712)
+  - [Itou Shizuka - Honjitsu, Mankai Watashi iro! (TV Size) \[TerisKey's Insane\]](https://osu.ppy.sh/beatmapsets/15381#fruits/58440)
+  - [Big Bang - Haru Haru (Nexy) \[Nuts\]](https://osu.ppy.sh/beatmapsets/7354#fruits/31730)
+  - [Max Coveri - Running in the 90's (Gabi) \[Alazy\]](https://osu.ppy.sh/beatmapsets/18482#fruits/81498)
+  - [Nien - MiNd CoNTRoL (lesjuh) \[HardStyle\]](https://osu.ppy.sh/beatmapsets/7612#fruits/32438)
+  - [NicoNicoDouga - Night Of Commands feat. YuGiOh (vytalibus) \[Start!!!!\]](https://osu.ppy.sh/beatmapsets/4823#fruits/26397)
+  - [Nakagawa Shouko - Happily Ever After (dksslqj) \[Insane\]](https://osu.ppy.sh/beatmapsets/17550#fruits/62591)
+  - [Super Junior-M - Super Girl (Chinese Version) (JauiPlaY) \[Hard\]](https://osu.ppy.sh/beatmapsets/16084#fruits/90383)
+  - [sun3 - Higan Retour (saymun) \[Alazy\]](https://osu.ppy.sh/beatmapsets/14464#fruits/54371)
+  - [Kajiura Yuki - When Two Powers Collide (Lizbeth) \[Insane\]](https://osu.ppy.sh/beatmapsets/11864#osu/44983)
+- Tiebreaker
+  - **[Ryu\* - Bass 2 Bass (yangzd88) \[Another\]](https://osu.ppy.sh/beatmapsets/30296#osu/101727)**
+
+------------------------------------------------------------------------
+
+## Match Results
+Due to old multiplayer data purged, multiplayer data can't be accessed.
+
+### Finals
+
+| Group | Saturday, 2012-07-28 |  |  |
```

Comment:
```suggestion
| Group | Saturday, 2012-07-28 |  |  |
```
Don't know if this causes problem.
We usually use this format

```
28 July 2012:

| Group | Player 1 |  |  | Player 2 | Match link |
| :-: | --: | :-: | :-: | :-- | :-- |
| Championship | Player 1 | Player 1's score | Player 2's score | The match link |
```

Reply:
I guess I have to generate a format for 1v1 as it's a generalized one (from CIC13)